frontend/cypress/integration/01-organization_sign_up_spec.js

```diff
@@ -1,4 +1,13 @@
-const { loginHooks } = require("../support");
+const {
+  loginHooks,
+  generateFacility,
+  generateOrganization,
+  generateUser,
+} = require("../support");
+
+const facility = generateFacility();
+const organization = generateOrganization();
+const user = generateUser();
```

Comment:
Good stuff 👍 At some point, I'd like to add some mechanism to clean up these randomly created orgs from staging after the tests past, to avoid filling up our Okta instance with a ton of junk groups. But I don't think it should block this PR. Can you create a ticket for that work so we don't lose track?

Reply:
I'll write up those tickets before merging this PR.

Reply:
https://app.zenhub.com/workspaces/simplereport-eng-health-61127bc8b61cae0013456681/issues/cdcgov/prime-simplereport/3630